runner.electron/main.js

```diff
@@ -1,9 +1,16 @@
 var app = require('electron').app;
 var BrowserWindow = require('electron').BrowserWindow;
+var pkg = require('./package.json');
 
 var mainWindow = null;
 var opts = %OPTS%;
 
+if (pkg && Array.isArray(pkg.commandLineSwitches)) {
+	pkg.commandLineSwitches.forEach(function(cliSwitch) {
+		app.commandLine.appendSwitch(cliSwitch);
```

Comment:
As far I see in https://github.com/electron/electron/blob/master/docs/api/chrome-command-line-switches.md it's also possible to pass some values to switches. Do you think we can use object instead of array to make this possible in this PR?

Reply:
Good idea, I'll add this.